doc/connection-backoff.md

```diff
@@ -24,9 +24,25 @@ ConnectWithBackoff()
          != SUCCESS)
     SleepUntil(current_deadline)
     current_backoff = Min(current_backoff * MULTIPLIER, MAX_BACKOFF)
-    current_deadline = now() + current_backoff
+    current_deadline = now() + current_backoff +
+      UniformRandom(-JITTER * current_backoff, JITTER * current_backoff)
+
 ```
 
+With specific parameters of
+INITIAL_BACKOFF = 20 seconds
```

Comment:
20 seconds seems a bit high for INITIAL_BACKOFF. How was this picked ? Are we specifying a timeout on TCP's connection attempt (because it has its own retries and exponential backoff, and hence would need to be truncated.)